hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/OMDBInsightEndpoint.java

```diff
@@ -88,15 +91,22 @@ public class OMDBInsightEndpoint {
   private static final Logger LOG =
       LoggerFactory.getLogger(OMDBInsightEndpoint.class);
   private final GlobalStatsDao globalStatsDao;
+  private ReconNamespaceSummaryManagerImpl reconNamespaceSummaryManager;
+  private Table<Long, NSSummary> nsSummaryTable;
```

Comment:
@ArafatKhan2198  Thanks for working on this patch, Pls do not expose the NSSummary table directly in endpoint class. Since you are already taking reference of namespaceSummaryManager and it already has ref of NSSummary table directly, use that and expose method for your operations.

Reply:
Done!